### ISSUE TITLE
Improper use of aria-expanded for drawer dialog

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-core#658]
### Subject of the issue

The Drawer dialog modal shouldn't have an `aria-expanded` state. 

> Note: The presence of the aria-expanded attribute indicates control. Avoid including it on elements that do not control the expanded state of other elements.

See [MDN note](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-expanded#:~:text=When%20the%20popup%20element%20is,should%20be%20set%20to%20true%20.&text=Note%3A%20The%20presence%20of%20the,expanded%20state%20of%20other%20elements) for reference.

For example, in the Accordion component, [`aria-expanded`](https://github.com/adaptlearning/adapt-contrib-accordion/blob/d98466c847543457efcaff0289d25f9bbfc85323/templates/accordion.jsx#L53) is applied to the `<button>` that controls the expand/collapse of the item content. This is the intended use.

This might just be legacy code prior to the Drawer changed to use `<dialog>` as `aria-expanded` isn't applied to the Notify `<dialog>`.

See [PR where <dialog> was introduced](https://github.com/adaptlearning/adapt-contrib-core/pull/601) for reference.

### Expected behaviour
`aria-expanded` should be removed from drawer `<dialog>` as this serves no purpose and it is not the intended use.
